app/src/pages/trace/SpanAnnotationForm.tsx

```diff
@@ -178,11 +178,6 @@ export function SpanAnnotationForm(props: SpanAnnotationFormProps) {
                 type="submit"
                 size="compact"
                 isDisabled={!isValid || !isDirty || isSubmitting}
-                onClick={() => {
-                  // TODO: This is a bit of a hack as the form is not working in a dialog for some reason
```

Comment:
the form seems to be submitting fine now, and this is causing double submission causing uniquness constraint errors in the db. Removing this fixes

### Before
![image (10)](https://github.com/user-attachments/assets/b9e40171-595c-4cf8-90c0-133e860124bb)


### After
![image](https://github.com/user-attachments/assets/cfc2067f-870e-4312-adf0-c717de5130cf)

Reply:
hmm, yeah this form submission in a dialog had me scratching my head. I can pull down tomorrow to investigate. I needed this on my machine but maybe you are right to remove.

Reply:
It was double submitting with, the safest thing if we're not sure if it double submits is to do the opposite of what i did and only do the onClick